client.html

```diff
@@ -7,14 +7,52 @@
         var gsegQS = "qs=w&qs=g&qs=t";
         var quantSegs = "a=E012365&a=E012367&a=E014326&a=E014365&a=E014383&a=E012633&a=E013060&a=E012551&a=E012730&a=E013714&a=E013568D073364D073365&a=E012712&a=E012369&a=E013630&a=E013488&a=E013048&a=E014343&a=E014329&a=E013067&a=E014179&a=E012725&a=E014298&a=E014293&a=E012389D074334&a=E012841D071917&a=E012775&a=E012390&a=E013869&a=E012697&a=E014204&a=E013685&a=E014224&a=E012449&a=E013151&a=E012360&a=E012731&a=E012692D073381&a=E013611D071920&a=E014178&a=E014171&a=E014148&a=E014147&a=E013075&a=E013011&a=E013334&a=E012387&a=E013547&a=E013657&a=E012370&a=E013818&a=E012741&a=E012724&a=E012476&a=E013806&a=E013763&a=E013759&a=E013393&a=E013711&a=E013554&a=E013463&a=E012351&a=E012479&a=E013509&a=E013490&a=E012362&a=E012377&a=E012380&a=E012386&a=E012684&a=E012729&a=E012763&a=E012819&a=E012820&a=E012822&a=E012823&a=E013123&a=E013169&a=E013368&a=E013442&a=E013460&a=E013464&a=E013497&a=E013510&a=E013529&a=E013551&a=E013559&a=E013560&a=E013589&a=E013644&a=E013447&a=E013707&a=E012574&a=E012849&a=E013724&"
     </script>
+
+
+    <!-- Sourcepoint vanilla CCPA impl. -->
+    <script>
+      (function () { var e = false; var c = window; var t = document; function r() { if (!c.frames["__uspapiLocator"]) { if (t.body) { var a = t.body; var e = t.createElement("iframe"); e.style.cssText = "display:none"; e.name = "__uspapiLocator"; a.appendChild(e) } else { setTimeout(r, 5) } } } r(); function p() { var a = arguments; __uspapi.a = __uspapi.a || []; if (!a.length) { return __uspapi.a } else if (a[0] === "ping") { a[2]({ gdprAppliesGlobally: e, cmpLoaded: false }, true) } else { __uspapi.a.push([].slice.apply(a)) } } function l(t) { var r = typeof t.data === "string"; try { var a = r ? JSON.parse(t.data) : t.data; if (a.__cmpCall) { var n = a.__cmpCall; c.__uspapi(n.command, n.parameter, function (a, e) { var c = { __cmpReturn: { returnValue: a, success: e, callId: n.callId } }; t.source.postMessage(r ? JSON.stringify(c) : c, "*") }) } } catch (a) { } } if (typeof __uspapi !== "function") { c.__uspapi = p; __uspapi.msgHandler = l; c.addEventListener("message", l, false) } })();
+    </script>
+
+    <script>
+      window._sp_ccpa = {
+          config: {
+              mmsDomain: 'https://message.sp-prod.net',
```

Comment:
Would we want to use our own domain?

```suggestion
              mmsDomain: 'https://consent.theguardian.com',
```

Reply:
This file's only for local testing, better keep it as close to standard as possible.